src/components/map/MapService.js

```diff
@@ -82,14 +82,34 @@ goog.require('ga_urlutils_service');
         Object.defineProperties(olLayer, {
           visible: {
             get: function() {
-              return this.getVisible();
+              return this.appvisible;
```

Comment:
yes. basically, what the user says visiblity should be. could be renamed though.